### PR TITLE
Docs: Distinguish examples for space-before-blocks

### PR DESCRIPTION
**Pull request by Krinkle**
**What is the purpose of this pull request? (put an "X" next to item)**

```
[x] Documentation update
```

**Please check each item to ensure your pull request is ready:**

- [x] I've read the [pull request guide](http://eslint.org/docs/developer-guide/contributing/pull-requests)

**What changes did you make? (Give an overview)**

This makes phrasing more consistent with other rules, and helps the site publisher set the appropriate icons and background colors for these examples.

### CODE REVIEW

File: docs/rules/space-before-blocks.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ class Foo{
 }
 ```
 
-The following patterns are not considered problems:
+Examples of **correct** code for this rule with the "always" option.
```
Comment (platinumazure) [inline docs/rules/space-before-blocks.md]: You end with a colon on line 30 and a period here. I like the colon better, mind switching?

File: docs/rules/space-before-blocks.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ try {} catch(a) {}
 
 ### "never"
 
-The following patterns are considered problems:
+Examples of **incorrect** code for this rule with the "never" option.
```
Comment (platinumazure) [inline docs/rules/space-before-blocks.md]: You end with a colon on line 30 and a period here. I like the colon better, mind switching?

File: docs/rules/space-before-blocks.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ for (;;) {
 try {} catch(a) {}
 ```
 
-The following patterns are not considered problems:
+Examples of **correct** code for this rule with the "never" option.
```
Comment (platinumazure) [inline docs/rules/space-before-blocks.md]: You end with a colon on line 30 and a period here. I like the colon better, mind switching?

**Comment by platinumazure** [comment]
Just some nitpicks about punctuation, otherwise looks great. Thanks so much for doing this!

**Comment by Krinkle** [comment]
@platinumazure Done (edited online). Feel free to squash when merging.

**Comment by kaicataldo** [comment]
LGTM. Thanks for contributing to ESLint!

**Pull request merged**